editor/src/components/canvas/canvas-strategies/post-action-options/navigator-reparent.ts

```diff
@@ -173,13 +175,15 @@ export const PropsReplacedNavigatorReparentPostActionChoice = (
 function adjustIntendedCoordinatesForGroups(
   jsxMetadata: ElementInstanceMetadataMap,
   pathTrees: ElementPathTrees,
+  allElementProps: AllElementProps,
   reparentTargetPath: ElementPath,
   intendedCoordinates: CanvasPoint,
   element: ElementInstanceMetadata | null,
 ): CanvasPoint {
-  const reparentTargetParentIsGroup = treatElementAsGroupLike(
+  const reparentTargetParentIsGroup = allowGroupTrueUp(
```

Comment:
I think here we should be using the more optimistic new treatElementAsGroupLike, because this code tries to make the freshly reparented element be a good group citizen, and that should be the desired behavior even if the group is currently broken

Reply:
Fixed in c381620da.